sql/sql_class.cc

```diff
@@ -5364,3 +5364,50 @@ bool THD::is_current_stmt_binlog_disabled() const
   return (!(variables.option_bits & OPTION_BIN_LOG) ||
           !mysql_bin_log.is_open());
 }
+
+/** Gets page fragmentation statistics. Assigns zeros to stats if thd is
+NULL.
+@param[in]  thd   the calling thread
+@param[out] stats a reference to fragmentation statistics to fill */
+void thd_get_fragmentation_stats(const THD *thd,
+                                 fragmentation_stats_t& stats)
```

Comment:
I don't mind but current style would be to use pointer here.

Reply:
garabage changed to "deleted_recs"